blobstore/cli/common/fmt/fmt_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 The CubeFS Authors.
```

Comment:
The year of the newly added file is the current year, 2023.

Reply:
done